src/main/java/me/zeroeightsix/kami/module/modules/render/Search.java

```diff
@@ -270,6 +279,41 @@ private boolean shouldUpdate() {
         return false;
     }
 
+    private void lineToBlock(BlockPos pos, int colour, float alpha){
+        Vec3d eyes = new Vec3d(0.0, 0.0, 1.0)
+                .rotatePitch((float) -Math.toRadians(mc.player.rotationPitch))
+                .rotateYaw((float) -Math.toRadians(mc.player.rotationYaw));
+
+        float red = ((float) (colour >> 16 & 0xFF)) / 255;
+        float green = ((float) (colour >> 8 & 0xFF)) / 255;
+        float blue = ((float) (colour & 0xFF)) / 255;
+
+        drawLineFromPosToPos(eyes.x, eyes.y  + mc.player.getEyeHeight(), eyes.z, pos.x + 0.5 -  mc.getRenderManager().renderPosX, pos.y + 0.5 -  mc.getRenderManager().renderPosY, pos.z + 0.5 -  mc.getRenderManager().renderPosZ, red, green, blue, alpha);
+    }
+
+    private void drawLineFromPosToPos(double posx, double posy, double posz, double posx2, double posy2, double posz2, float red, float green, float blue, float opacity) {
```

Comment:
Could this go in render manager or KamiTesselator please? Also, the tracers module already has a from pos to player iirc

Reply:
Yeah should I update tracers too while im at it or just leave it be?